python/cudf/cudf/testing/io.py

```diff
@@ -76,3 +79,10 @@ def get_next_regression_params(self):
         self._idx += 1
         self._current_params = copy.copy(param)
         return dtypes_meta, num_rows, num_cols, seed
+
+    def get_rand_params(self, params):
```

Comment:
Is this a public class meant to be used by end-user? If so, we should add docstrings to its `__init__` and its public methods.

If not, maybe we should make it an underscore class `_IOFuzz`, because `cudf.testing` is a public module.

Reply:
Ditto for the `cudf.testing.main` module.

Reply:
Moved the entire python fuzz testing framewok and related test to `_fuzz_testing` instead..